index.html

```diff
@@ -0,0 +1,172 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="assets/css/mobile.css" media="screen and (max-width: 768px)">
+    <link rel="stylesheet" href="assets/css/desktop.css" media="screen and (min-width: 768px)">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap">
+    <title>cybersecuritysummit</title>
+</head>
+<body>
+    <!-- header-section -->
+<header class="main-header-section">
+  <nav class="nav-bar1">
+    <ul class="desktop-upper">
+      <li><i class="fa fa-facebook"></i></li>
+      <li><i class="fa fa-twitter"></i></li>
+      <li>English</li>
+      <li>My Page</li>
+      <li>Logout</li>
+    </ul>
+  </nav>
+    <nav class="nav-bar">
+      <div class="mobile-menu"><i id="fa" class="fa fa-bars fa-2x" aria-hidden="true"></i></div>
+      <ul class="desktop-nav">
+        <div class="logo-wrapper">
+          <img src="assets/image/logo-for-capstone.png" class="Nig-logo" alt="Nigeria-coat-of-arm-logo">
+          <ul class="logo">
+          <li id="ils">ILS Conference on <br> Cybersecurity 2021</li>
+          </ul>
+     </div>
+        <li><a id="about" href="about.html">About</a></li>
+        <li><a href="">Program</a></li>
+        <li><a href="">Join</a></li>
+        <li><a href="">Sponsor</a></li>
+        <li><a href="">News</a></li>
+        <div class="Ils">ILS Conference</div>
+    </ul>
+    </nav>
+    <div class="header-body">
+    <p class="header-text">"Hello! Compatriots"</p>
+    <div class="text-logo">
+    <h1 class="header-main-text">2021 International <br> Legislative Stakeholders <br> Conference on <br> Cybersecurity.</h1>
+    <img src="assets/image/cyber-logo.jfif" id="cyber-logo" alt="">
+    </div>
+    <div class="ict-text">
+        <p>ICT revolution has brought about expanded benefits to all human endeavours that cannot be over-emphasized. The growth of digital technology has brought about improved performance in all aspects of society. It is an economic development pillar guaranteeing a competitive advantage.</p>
+    </div>
+    <div class="date-container">
+        <ul class="header-date">
+            <li><i class="fa fa-calendar-check-o" aria-hidden="true"></i></li>
+            <li class="date">2021.12.7(TUE) ~ 9(THU) (GMT +1)</li>
+        </ul>
+    </div>
+    <p class="address-text">@ Zeus paradise hotel, 1 Domenico Gitto, off Mabushi-Jabi Road, Mabushi Abuja.</p>
+</div>
+
+<aside class="nav-menu">
+  <button class="aside-close">X</button>
+  <ul class="mobile-nav">
```

Comment:
- [ ] When the aside menu is open, users can still scroll the body section of the website, this is how it looks: 
![image](https://user-images.githubusercontent.com/77038610/147233147-1355b352-dc72-458f-911b-ec6db7d5b2b9.png)

To avoid this situation, you can easily set the `body{ overflow: hidden }` when users click on hamburger button,  `body{ overflow: auto; }` when users close the aside menu

Reply:
Body set to overflow and hidden.